### DISCUSSION TITLE
v2.3.110 Fleetdm Enable software inventory on hosts

### DISCUSSION THREAD

**Discussion by kevinelwell** [Security-Onion-Solutions/securityonion#7643] [Unsupported Versions] [answered]
I am new-ish to SO. I installed the latest version, install osquery agent on one vm and was trying to figure out how to enable software inventorying. Can anyone please point me in the right direction?

I tried running fleet and fleetctl and neither binary was found.

**Comment by OhHappyDagger** [comment]
It looks like this was intentionally disabled in [this commit](https://github.com/Security-Onion-Solutions/securityonion/commit/c4babf22d6255ce953d2696c69238068741e9d97). I've tried modifying the so-fleet Docker image but the UI still shows that feature is disabled. (Any changes to this would be overwritten by salt anyways) This is something I would like to see implemented as well.

**Comment by defensivedepth** [comment, marked as answer]
We have disabled this feature by default until it matures a bit more.

Please see my post over in the FleetDM repo, which outlines the issues:  https://github.com/fleetdm/fleet/issues/3081#issuecomment-982998505

**Comment by talabs** [comment]
Understanding that this is disabled by default and managed by salt how would you go about enabling it to get familiar with it's operation even though there would be false positives?

I am running SO 2.3.140 and trying to familiarize myself with its interface and alerting tools. I know that software inventory is disabled by default in /opt/so/conf/fleet/packs/osquery-config.conf and if changed there it is overwritten by salt. Is it possible to edit an sls file to tell salt to modify enable_software_inventory to true?

I have never worked with salt prior to this so I'm still learning.

**Comment by isaacgolding** [comment]
If you still want the fleet software inventory enabled here is a link to your only option.  https://github.com/Security-Onion-Solutions/securityonion/discussions/7844#discussioncomment-2647835

However I'll warn you that if you reinstall SO at a later time and date you will break the link between it and the agent that provides the data to the fleet interface.   Every client computer will need to have the fleet agent removed and reinstalled.   After going through this a couple times I removed the agent from all but 2 computers and am just using those two as my test subjects for now.

**Reply by talabs** [reply]
Understood, right now it's only my admin PC that is going to be part of this for testing. Thank you for the link.
